src/Entity/Log/Event.php

```diff
@@ -65,7 +65,7 @@ public function getId(): ?string
     }
 
     /**
-     * @return ?class-string<AbstractEvent>
+     * @return ?string
```

Comment:
Why was this type hint generalized?

Reply:
top of my head, Stan asked me to.

Reply:
Is there a way to change that? I feel that a narrowed type definition is preferable over a more general one.